drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -167,6 +168,22 @@ def mci_id
     ac_hmis_mci_ids.to_a.min_by(&:id)&.value
   end
 
+  def update_mci_client
+    return unless ac_hmis_mci_ids.exists?
+    return unless [
+      'FirstName',
+      'LastName',
+      'MiddleName',
+      'DOB',
+      'SSN',
+      'Gender',
+      'GenderNone',
+      'GenderOther',
+    ].any? { |field| previous_changes&.[](field) }
+
+    HmisExternalApis::AcHmis::UpdateMciClientJob.perform_later
```

Comment:
Don't we need to pass the client?
```suggestion
    HmisExternalApis::AcHmis::UpdateMciClientJob.perform_later(client: self)
```